src/gaurav_garg/PatternPrintAssignment79.java

```diff
@@ -0,0 +1,38 @@
+/*
+1
+22
+333
+4444
+
+*
+##
+***
+####
+*****
+ */
+package gaurav_garg;
+
+public class PatternPrintAssignment79 {
+
+	public static void main(String[] args) {
+		int k = 1;
+		for (int i = 0; i < 4; i++) {
```

Comment:
Make code generic so if you want to print for 10 or 2 rows then also it is possible